schema/deploy/shipping/views.sql

```diff
@@ -38,4 +38,30 @@ grant select
     on shipping.presence_absence_result_v1
     to "incidence-modeler";
 
+
+create or replace view shipping.presence_absence_result_v2 as
+
+    select sample.identifier as sample,
+           target.identifier as target,
+           present,
+           organism.lineage as organism,
+           presence_absence.details ->> 'device' as device
+
+    from warehouse.sample
+    join warehouse.presence_absence using (sample_id)
+    join warehouse.target using (target_id)
+    left join warehouse.organism using (organism_id)
+    where target.control = false;
+
+comment on view shipping.presence_absence_result_v2 is
+    'View of warehoused presence-absence results for modeling and viz teams';
+
+revoke all
+    on shipping.presence_absence_result_v2
+  from "incidence-modeler";
+
+grant select
+    on shipping.presence_absence_result_v2
+    to "incidence-modeler";
+
```

Comment:
If the only change we're making to this view is adding a new column, I don't think it's necessary to bump the version number.

Reply:
The only reason I bumped it was because `sqitch revert` won't work, but if we're fine with that then I can change back to v1

Reply:
Ah do you mean `sqitch revert` won't work because we added a new column? I recall running into a similar problem. I _think_ `sqitch revert` should work if you drop the view then re-create it. I'd prefer to keep our versioning system of only bumping the revision number if we're making breaking changes to the view.

Reply:
I can't drop the view because there are other views dependent on it

Reply:
I'm going to add the COVID-19 results on top of v2, so I'm fine keeping it as is.